Limelight/Input/StreamView.m

```diff
@@ -49,11 +52,25 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
         UITouch *touch = [[event allTouches] anyObject];
         touchLocation = [touch locationInView:self];
         touchMoved = false;
+        dragTimer = [NSTimer scheduledTimerWithTimeInterval:2.0
```

Comment:
Can we get another touchesBegan before touchesEnded? We don't want to leave a timer dangling.